toolchain/check/import_ref.cpp

```diff
@@ -1606,9 +1624,6 @@ static auto AddClassDefinition(ImportContext& context,
 
   new_class.complete_type_witness_id = complete_type_witness_id;
 
-  new_class.scope_id = context.local_name_scopes().Add(
-      new_class.first_owning_decl_id, SemIR::NameId::None,
-      new_class.parent_scope_id);
   auto& new_scope = context.local_name_scopes().Get(new_class.scope_id);
```

Comment:
We're setting the other properties of `new_scope` here; I wonder if it'd make sense to call `new_scope.Set(...)` here too?

Reply:
I guess it is just `AddNameScopeImportRefs`. Nonetheless I think it makes sense to put all the code that builds the imported scope together rather than splitting it across two functions.

Reply:
oh, and the comment about why we do it in phase 3 didn't seem suitable inside `AddClassDefinition`, since it doesn't talk about phases.

Reply:
I think eventually we should sink the `Set` call into `AddNameScopeImportRefs` and rename it (maybe something like `ImportNameScope`?), but we'll need to update the other callers to follow the same pattern first.